providers/src/airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -392,6 +392,7 @@ class EcsRunTaskOperator(EcsBaseOperator):
     :param deferrable: If True, the operator will wait asynchronously for the job to complete.
         This implies waiting for completion. This mode requires aiobotocore module to be installed.
         (default: False)
+    :param container_name: The name of the container to fetch logs from. If not set, the first container is used.
```

Comment:
Do you mind here and below if you move this param up to be grouped with the other ECS API related params?

Reply:
Done in https://github.com/apache/airflow/pull/46152/commits/b8ba685fd2d89e3f139f587b68256c886df74512